src/App.css

```diff
@@ -1,3 +1,8 @@
+.Nav-bar {
+  border: 1px solid red;
```

Comment:
- [ ] I think you have forgotten this red border over here, please think of removing it before